src/raygun.rum/index.js

```diff
@@ -1203,7 +1206,7 @@ var raygunRumFactory = function (window, $, Raygun) {
 
     function sanitizeNaNs(data) {
       for (var i in data) {
-        if (isNaN(data[i]) && typeof data[i] !== 'string') {
+        if (data[i] === "NaN") {
```

Comment:
Could change to check string and nan
```suggestion
        if (data[i] === "NaN" || data[i] === NaN) {
```